onnxscript/evaluator.py

```diff
@@ -12,6 +12,8 @@
 
 import numpy as np
 import onnx
+import torch
```

Comment:
It means onnxscript takes a dependency on torch. Maybe that would be worth adding this in a separate folder.

Reply:
In the future this code will live in torch so I think we are good? In reverse, torch will take dependency on onnxscript.

Reply:
breaking it out to make it clear

Reply:
> In the future this code will live in torch

Agree it'd be clearer for all such code to live in a dedicated directory.

Reply:
Done